test/cmd/basicresources.sh

```diff
@@ -84,8 +84,9 @@ os::cmd::expect_failure_and_text 'oc get' 'deploymentconfig'
 os::cmd::expect_success_and_text 'oc get all --loglevel=6' 'buildconfigs'
 os::cmd::expect_success_and_text 'oc explain pods' 'Pod is a collection of containers that can run on a host'
 os::cmd::expect_success_and_text 'oc explain pods.spec' 'SecurityContext holds pod-level security attributes'
-os::cmd::expect_success_and_text 'oc explain deploymentconfig' 'a desired deployment state'
-os::cmd::expect_success_and_text 'oc explain deploymentconfig.spec' 'ensures that this deployment config will have zero replicas'
+# TODO figure out how to delegate the swagger API calls somehow
+# os::cmd::expect_success_and_text 'oc explain deploymentconfig --api-version=apps.openshift.io/v1' 'a desired deployment state'
```

Comment:
If I'm looking correctly this should be addressed in https://github.com/openshift/origin/pull/15564/commits/3aef22f8282112a59f8fd85495e198ff8f0bdab4, which is dropping the openapi disablement, isn't it?